tmail-backend/apps/postgres/src/doc/release-instructions/1.0.17/index.md

```
@@ -0,0 +1,82 @@
+
+Adapt the signatures to your needs. 
+
+### RabbitMQ event bus partitioning (Optional)
+
+Allow to enable partitioning on RabbitMQ event bus queues, to better leverage CPU resource on the RabbitMQ side.
+
+Note that it is only possible if you are using the RabbitMQ and Redis event bus with Twake Mail.
+
+If you wish to enable it, in `rabbitmq.properties`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)